manager-parent/manager-core/src/main/resources/application.properties

```diff
@@ -153,9 +87,6 @@ event-bridge.dns.aws.route53.secret-access-key=${AWS_SECRET_ACCESS_KEY}
 event-bridge.dns.ingress.override.hostname=${EVENT_BRIDGE_DNS_INGRESS_OVERRIDE_HOSTNAME:localhost}
 
 # LOGGING
-# Use unstructured logging in DEV/test
-%dev.quarkus.log.console.json=false
-%test.quarkus.log.console.json=false
```

Comment:
I moved the `TEST` override to `src/main/test/resources/application.properties` for consistency with other test overrides.